src/lock_manager.h

```diff
@@ -25,6 +25,7 @@
 #include <string>
 #include <functional>
 
+
```

Comment:
Hi. Is this an empty line added accidentally? It seems that it has nothing to do with your pr.
```suggestion
```

Reply:
sorry，i don't know ..... 
maybe because i run check locally